### PR TITLE
Add label resolver blog post

### PR DESCRIPTION
**Pull request by Noviny**

### CODE REVIEW

**Comment by JedWatson** [comment]
Read through it @Noviny and it looks great 👍 

Ready to go when we have screenshots in

**Review by JedWatson — APPROVED** [review]

File: docs/blog/better-ux-with-custom-labels.md [hunk 1]
```diff
@@ -134,7 +138,7 @@ const Post = {
     // and the rest of the fields too
   },
   labelResolver: async item => {
-    const { data } = await keystone.executeGraphQL({
+    const { data } = await Keystone.executeGraphQL({
```
Comment (Noviny) [inline docs/blog/better-ux-with-custom-labels.md:141]: @JedWatson  this conflicts with what we do in all our docsssss!

Comment (JedWatson) [reply docs/blog/better-ux-with-custom-labels.md:141]: haha whoops, that got caught in the search/replace for the case change. Wanna change it back?

**Comment by JedWatson** [comment]
@Noviny I made some tweaks directly -

- Consistent casing of "keystone" as "Keystone" and "admin UI" as "Admin UI"
- Rewrote the "What is the Label?" section
- Slight changes to tone of voice, simplified language
- Fixed the slug for consistency and clearer url structure

**Pull request merged**